.travis.yml

```diff
@@ -6,44 +6,23 @@ notifications:
 
 env:
   global:
-    - NENGO="false"
-    - STATIC="false"
-    - DOCS="false"
+    - PYTHON="3.5.2"
 
 matrix:
   include:
-    - env: PYTHON="3.5.2"
-    - env: PYTHON="3.5.2" NENGO="true"
-    - env: PYTHON="3.5.2" STATIC="true"
-    - env: PYTHON="3.5.2" DOCS="true"
+    - env: MODE="test"
+    - env: MODE="static"
+    - env: MODE="docs"
 
 before_install:
+  - chmod +x .ci/*.sh
```

Comment:
We should mark all the scripts as executable in git rather than as part of the build process I think... can do this quickly in the merge.